ros2_control_demo_robot/launch/diffbot_system.launch.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 ROS2-Control Development Team (2020)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+import xacro
+
+
+def generate_launch_description():
+
+    # Get URDF via xacro
+    robot_description_path = os.path.join(
+        get_package_share_directory('ros2_control_demo_robot'),
+        'description',
+        'diffbot_system.urdf.xacro')
+    robot_description_config = xacro.process_file(robot_description_path)
+    robot_description = {'robot_description': robot_description_config.toxml()}
+
+    diffbot_diff_drive_controller = os.path.join(
+        get_package_share_directory('ros2_control_demo_robot'),
+        'controllers',
+        'diffbot_diff_drive_controller.yaml'
+        )
+
+    return LaunchDescription([
+      Node(
+        package='robot_state_publisher',
+        executable='robot_state_publisher',
+        name='robot_state_publisher',
+        output='screen',
+        parameters=[robot_description]),
+
+      Node(
+        package='joint_state_publisher',
```

Comment:
Then you probably also need a Joint State Controller.

Reply:
Actually I have a problem with Joint State Controller. I don't know if it is a bug or if I am missing something, but when echoing joint_states topic the correct joint state values is not been constantly published. It is actually been mixed with zeros and .nan publishes. 
This behavior also happen with rrbot example and in RViz I can see the robot state oscillating between 0 and some other state.
Do you know what could be happening @destogl?

Reply:
No idea. For me is working great for multiple robots...